src/app/components/core/control-dispatcher/pipelines/add-pipeline/add-control-pipeline.component.html

```diff
@@ -221,7 +221,7 @@
                             <div *ngIf="selectedDestinationType?.name && !(selectedDestinationType.name === 'Any' || selectedDestinationType.name === 'Broadcast')"
                               class="field is-horizontal">
                               <div class="field-label is-small">
-                                <label class="label is-small">Name</label>
+                                <label class="label lbl-child">Name</label>
```

Comment:
this is okay;

Reply:
You mean here, for child label I should not use `lbl-child` class i.e. `font-size: 0.85rem` ?